pkg/apis/monitoring/v1beta1/alertmanager_config_types.go

```diff
@@ -908,6 +908,9 @@ type MSTeamsConfig struct {
 	// Message body template.
 	// +optional
 	Text *string `json:"text,omitempty"`
+	// Message summary template.
+	// +optional
+	Summary *string `json:"summary,omitempty"`
```

Comment:
Hey thanks to handle this @heliapb
Since this is a feature only available on AM greater than or equal to 0.27 I think worth add this validation to `checkMSTeamsConfig` available [here](https://github.com/prometheus-operator/prometheus-operator/blob/35532ddf64483d2468b07bddc23bac329302d279/pkg/alertmanager/operator.go#L1615).

Reply:
Hi @nicolastakashi added a validation, could you review?